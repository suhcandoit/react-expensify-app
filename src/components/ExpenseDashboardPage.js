import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseFilters from './ExpenseFilters';

const ExpenseDashboardPage = () => (
    <div>Expense Dashboard
        <ExpenseFilters />
        <ExpenseList />
    </div>
);

export default ExpenseDashboardPage;