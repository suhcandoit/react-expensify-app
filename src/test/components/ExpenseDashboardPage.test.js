import React from 'react';
import { shallow } from 'enzyme';
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage';

test('Should render ExpenseDashboard page', () => {
    const wrapper = shallow(<w />)
    expect(wrapper).toMatchSnapshot()
})