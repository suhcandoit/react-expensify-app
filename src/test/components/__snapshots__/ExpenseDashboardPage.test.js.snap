// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ExpenseDashboard page 1`] = `
<div>
  Expense Dashboard
  <ConnectFunction />
  <ConnectFunction />
  <ConnectFunction />
</div>
`;
